consul_template/templates/consul-template.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Consul Template
+Requires=docker.service
+After=docker.service
+
+[Service]
+Restart=always
+ExecStartPre=-/usr/bin/docker stop consul-template
```

Comment:
Correct, if `docker stop` fails, the error is ignored and we move on to the next command. Stopping and removing the container will fail if the container doesn't exist.